### DISCUSSION TITLE
All nulls for the emails column

### DISCUSSION THREAD

**Discussion by alabora** [OvertureMaps/data#141] [Discussion]
Hi! I'm exploring the new data, thanks for the release. However, I seem to have found a problem.

I'm getting all nulls for the emails field in the Places database, I can't see what I might be doing wrong. Is this really it?

**Comment by jwass** [comment]
@alabora Thanks for reaching out. I think there is indeed an issue. We'll look into it and update here.

**Reply by ckaktena** [reply]
Hi jwass, do You solve that problem? When can we expected emails fill.
